internal/cmd/commands/targetscmd/funcs.go

```diff
@@ -76,6 +79,16 @@ func extraSynopsisFuncImpl(c *Command) string {
 		}
 		return wordwrap.WrapString(fmt.Sprintf("%s a target", in), base.TermWidth)
 
+	case "set-static-address", "remove-static-address":
+		var in string
+		switch {
+		case strings.HasPrefix(c.Func, "set"):
+			in = "Set the static address for"
+		case strings.HasPrefix(c.Func, "remove"):
+			in = "Remove the static address from"
+		}
+		return wordwrap.WrapString(fmt.Sprintf("%s a target.", in), base.TermWidth)
+
```

Comment:
What do you think about
```suggestion
	case "set-static-address":
		return wordwrap.WrapString("Set the static address for a target.", base.TermWidth)
	case "remove-static-address":
		return wordwrap.WrapString("Remove the static address from a target.", base.TermWidth)

```